### ISSUE TITLE
symlinks are not created on Windows10

### ISSUE THREAD

**Issue by fralbo** [oscarpilote/Ortho4XP#31]
As far as I understood [here](https://forums.x-plane.org/index.php?/forums/topic/157147-how-to-create-a-symlink-in-v130/), v1.30 automatically generates symlink in the X-Plane's `Custom Scenery`.
Well not sure if it's a very good idea compare to the 1.20b feature but it's not the problem here.

I've defined my X-Plane's `Custom Scenery` path in the config and I've generated several tiles but no symlink are generated.

**Comment by d41k4n** [comment]
As was already stated in the linked [forum topic](https://forums.x-plane.org/index.php?/forums/topic/157147-how-to-create-a-symlink-in-v130/&do=findComment&comment=1481051) you need to make sure the required conditions are met for this to work i.e.the DSF file was created at the expected location according to your "Base folder" setting. Did you?

Also, you can verify if the message pane contains a line with the message "Activating DSF" (or similar) when step 3 finishes. Can you confirm this was the case when you created your tiles?

**Comment by fralbo** [comment]
So, I think conditions are met:
![image](https://i.imgur.com/atediWU.png)
and the buiilt tile `zOrtho4XP_+48+002` contains dds files but no link and no "Activating DSF" or similar.

only `ERROR: Providers/O4_Custom_URL.py contains invalid code. The corresponding providers won't probably work.` but I don't think there is any relationship with the problem.

**Comment by d41k4n** [comment]
Assuming you left "Base Folder" empty on the main windows and your tile completed successfully there should be a file ./Tiles/zOrtho4XP_+48+002/Earth nav data/+40+000/+48+002.dsf in your Ortho4XP home. If not, that explains why creating a symlink doesn't work for that tile.

**Comment by fralbo** [comment]
Yes I have the dsf file. but not the symlink.

What I wasn't expecting for is that I manually created the symlink with Symbolic Link Creator and the map shows a dark tile corresponding to that link.

**Comment by d41k4n** [comment]
Where did you create the shortcut? Did the tile display on the map at all before you manually created the shortcut? Confirm your "Base Folder" value is empty and you have saved the config.

**Comment by d41k4n** [comment]
I just tested on my end (Win 7) using the latest master and I'm unable to reproduce the described behavior. Toggling shortcuts from the map using CTRL+LMB works fine. The tile color will darken after the shortcut is created. However, I noticed some tiles seem color-coded differently (yellow-green-ish vs yellow-blue-ish) - not sure what this means (but it doesn't make a difference regarding shortcuts).

In my case I left "Based Folder" empty, set Custom Scenery to "D:/X-Plane 11/Custom Scenery/" in the config window (leaving out the trailing path delimiter made no difference) and hit "Apply" and "Write App Cfg". From there I proceeded to the tile map and was able to create a shortcut for any highlighted tile successfully. I made sure the "Tiles" folder in Ortho4XP's home dir is an actual folder e.g. not a shortcut or junction.

If the tile is not highlighted with a black frame that means there was no DSF found. If the tile is highlighted but without a label (i.e. "BI 17") that means there was no tile .cfg found.

I don't see any reason why this shouldn't work on Win 10.
